src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -39,7 +39,7 @@ const sendErrorMetricAndAnalytics = (jiraHost: string, errorCode: ErrorResponseC
 	}, {
 		jiraHost,
 		...errorCodeAndStatusObj
-	});
+	}).catch(() => { /* ignore */ });
```

Comment:
Curious why are we are swallowing a error here?

Reply:
I believe we were previously swallowing this error because the promise was never awaited.  This is a little more explicit about it.  I should probably have logging....but the underlying functions are already logging on throws and silently passing up undefined.